anago

```diff
@@ -756,11 +761,6 @@ get_build_candidate () {
   # has been created first or warn.
   # This is to ensure that branchff can continue to be used as needed.  User can
   # override here.
-  if [[ ! $JENKINS_BUILD_VERSION =~ ${VER_REGEX[release]} ]]; then
-    logecho "Invalid format: $JENKINS_BUILD_VERSION"
-    return 1
-  fi
-
   # For master branch alpha builds, ensure we've released the previous .0 first
   if [[ -z "$PARENT_BRANCH" && -n ${RELEASE_VERSION[alpha]} ]]; then
     latest_official=${BASH_REMATCH[1]}.$((${BASH_REMATCH[2]}-1))
```

Comment:
Yeah, really.  Keeping that whole thing as a unit now in branchff_sanity_check().